js/page-specific/swiftype-search-results.js

```diff
@@ -15,21 +15,15 @@ baltimoreCounty.pageSpecific.swiftypeSearchResults = (function swiftypeSearchRes
 		}
 
 		var searchTerms = cleanedSearchTerm.split('+');
-		for (var i = 0; i < searchTerms.length; i += 1) {
+		for (var i = 0, len = searchTerms.length; i < len; i += 1) {
 			var term = decodeURIComponent(searchTerms[i]);
-			term = term
-				.replace(/\?/g, '')
-				.replace(/#/g, ' ')
-				.replace(/&/g, 'and')
-				.replace(/\//g, ' ')
-				.replace(/\\/g, ' ');
+			term = term.replace(/[&?#\\/]/g, ' ')
+				.replace(/&/g, 'and').trim();
```

Comment:
The second replace will never succeed, since you're replacing "&" in the first replace.